docs/website/docs/tutorial/rest-api.md

```diff
@@ -0,0 +1,320 @@
+- `append`: Appends the data to the destination table. This is the default.
+- `replace`: Replaces the data in the destination table with the new data.
+- `merge`: Merges the new data with the existing data in the destination table based on the primary key.
+
+### Replacing the data
+
+In our case, we don't want to append the data every time we run the pipeline. Let's start with the simpler `replace` write disposition.
+
+To change the write disposition to `replace`, update the `resource_defaults` configuration in the `rest_api_pipeline.py` file:
+
+```py
+...
+pokemon_source = rest_api_source(
+    {
+        "client": {
+            "base_url": "https://pokeapi.co/api/v2/",
+        },
+        "resource_defaults": {
+            "endpoint": {
+                "params": {
+                    "limit": 1000,
+                },
+            },
+            "write_disposition": "replace", # Setting the write disposition to `replace`
+        },
+        "resources": [
+            "pokemon",
+            "berry",
+            "location",
+        ],
+    }
+)
+...
+```
+
+Run the pipeline again with `python rest_api_pipeline.py`. This time, the data will be replaced in the destination table instead of being appended.
+
+### Merging the data
+
+When you want to update the existing data as new data is loaded, you can use the `merge` write disposition. This requires specifying a primary key for the resource. The primary key is used to match the new data with the existing data in the destination table.
+
+Let's update our example to use the `merge` write disposition. We need to specify the primary key for the `pokemon` resource and set the write disposition to `merge`:
+
+```py
+...
+pokemon_source = rest_api_source(
+    {
+        "client": {
+            "base_url": "https://pokeapi.co/api/v2/",
+        },
+        "resource_defaults": {
+            "endpoint": {
+                "params": {
+                    "limit": 1000,
+                },
+            },
+            # For the `berry` and `location` resources, we set
+            # the write disposition to `replace`
+            "write_disposition": "replace",
+        },
+        "resources": [
+            {
+                "name": "pokemon",
+                "primary_key": "id", # Setting the primary key for the `pokemon` resource
+                "write_disposition": "merge",
+            },
+            "berry",
+            "location",
+        ],
+    }
+)
+```
+
+## Loading data incrementally
+
+When working with some APIs, you may need to load data incrementally to avoid fetching the entire dataset every time and to reduce the load time. The API that support incremental loading usually provide a way to fetch only new or changed data (most often by using a timestamp field like `updated_at`, `created_at`, or incremental IDs).
```

Comment:
```suggestion
When working with some APIs, you may need to load data incrementally to avoid fetching the entire dataset every time and to reduce the load time. APIs that support incremental loading usually provide a way to fetch only new or changed data (most often by using a timestamp field like `updated_at`, `created_at`, or incremental IDs).
```